src/Clients/BaseClient.php

```diff
@@ -26,4 +48,43 @@ public function createFilterQuery(string $path, array $filters): string
 
         return "{$path}?{$queryString}";
     }
+
+    public function getCacheTtl(): int
+    {
+        return config('easyoffice-api.default_cache_liftime');
+    }
+
+    public function getCacheEnabled(): bool
+    {
+        return config('easyoffice-api.enable_cache', false);
+    }
+
+    public function generateCacheKey(string $endpoint, array $options = []): string
+    {
+        $cacheKey = "easyoffice__{$endpoint}";
+
+        foreach ($options as $key => $value) {
+            $cacheKey .= "__{$key}_{$value}";
+        }
```

Comment:
Nice approach for parameters. Awesome work!

Just a brain fart: maybe we should limit this to certain allowed parameters? And exclude other ones? Imagine Google Analytics parameters etc.

Reply:
As this is only used for api calls, I don't think google analytics parameters will be used?

I think it is beneficial so that the package does not have to be modified when we add additional parameters in the api.